dscntou/dscntou
# Add CoC page

(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies


## Description
**Add CoC page**
I add a code of conduct page and a link on navbar.

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to 'home'
2. Click on '行為準則' on navbar
3. See the page

## Expected behavior
CoC page can be visited by navbar and display correctly.

## Related Issue
Contents need to edit.
